datasheets/254_eng_tds--SdCard.md

```diff
@@ -0,0 +1,37 @@
+# MicroSD card breakout board+
+
+[Technical Details (Adafruit)](https://www.adafruit.com/product/254#technical-details)
+
+![image-20210829205636512](.254_eng_tds--SdCard/image-20210829205636512.png)
+
+![MicroSD card breakout board+](.254_eng_tds--SdCard/254-05.jpg)
+
+![additional product photo](.254_eng_tds--SdCard/254-03.jpg)
+
```

Comment:
Please apply meaningful names to the images.

Reply:
done